ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -213,13 +213,14 @@ MemoryConfig create_sharded_memory_config_from_parallel_config(
 
     uint32_t nhw_shape = tensor_shape[0] * tensor_shape[1] * tensor_shape[2];
     uint32_t nhw_padded = nhw_shape;
-    if(shard_scheme != TensorMemoryLayout::WIDTH_SHARDED) {
-        nhw_padded = round_up(nhw_shape, num_cores_nhw * tile_size);
-    }
+    // if(shard_scheme != TensorMemoryLayout::WIDTH_SHARDED) {
```

Comment:
If this is supposed to be removed, please remove instead of just commenting it out.

This goes for all the files in the PR.